src/testdrive/src/main.rs

```diff
@@ -110,10 +105,10 @@ async fn run(args: Args) -> Result<(), Error> {
             let credentials = provider
                 .credentials()
                 .await
-                .err_ctx("retrieving AWS credentials")?;
+                .expect("retrieving AWS credentials");
```

Comment:
```err.print_stderr(ci_output);``` is no longer used to print the AWS errors. Is that a problem? I am not sure which of the zillion ways to unpack Err would allow me to call print_stderr.

Reply:
Aw, I see, not ideal. Oh well. This seems fine for now!